examples/IRrecvDumpV2/IRrecvDumpV2.ino

```diff
@@ -131,6 +132,16 @@ void dumpACInfo(decode_results *results) {
     description = ac.toString();
   }
 #endif  // DECODE_KELVINATOR
+#if DECODE_MITSUBISHI_AC
+  Serial.println("Trying Mitchu AC:");
```

Comment:
Please remove the Serial.println()'s so it is consistent with the other decoders.

Reply:
Sorry, left here from debug rounds.